github_jira/cmd/creategithub.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/mattermost/mattermost-utilities/github_jira/github"
+	"github.com/mattermost/mattermost-utilities/github_jira/jira"
+	"github.com/spf13/cobra"
+)
+
+var createGithubCmd = &cobra.Command{
+	Use:     "creategithub",
+	Short:   "Create github issues from list of JIRA issue numbers",
+	Example: `  creategithub -u <user> -j <jira token> -g <github token> -r mattermost/mattermost-server -l 'Tech/Go,Help Wanted,Up For Grabs' 19977 12345`,
+	Args:    cobra.MinimumNArgs(1),
+	RunE:    createGithubCmdF,
+}
+
+func init() {
+	createGithubCmd.Flags().StringP("jira-token", "j", "", "The token used to authenticate the user against Jira.")
+	createGithubCmd.MarkFlagRequired("jira-token")
+	createGithubCmd.Flags().StringP("jira-username", "u", "", "Username of the user to get the ticket information.")
+	createGithubCmd.MarkFlagRequired("jira-username")
+	createGithubCmd.Flags().StringP("github-token", "g", "", "The token used to authenticate the user against Github.")
+	createGithubCmd.MarkFlagRequired("github-token")
+	createGithubCmd.Flags().StringP("repo", "r", "", "The repository which contains the issues. E.g. mattermost/mattermost-server")
+	createGithubCmd.MarkFlagRequired("repo")
+	createGithubCmd.Flags().StringSliceP("labels", "l", []string{}, "The labels to set to the issues")
+	createGithubCmd.MarkFlagRequired("labels")
+	createGithubCmd.Flags().Bool("dry-run", false, "Skip actually creating any tickets")
+	createGithubCmd.Flags().Bool("debug", false, "Dump debugging information.")
+
+	RootCmd.AddCommand(createGithubCmd)
+}
+
+func getStr(command *cobra.Command, name string) (string, error) {
+	str, err := command.Flags().GetString(name)
+	if err != nil {
+		return "", errors.New(fmt.Sprintf("invalid %s parameter", name))
```

Comment:
You can replace this and all the other similar cases with `fmt.Errorf`
```suggestion
		return "", fmt.Errorf("invalid %s parameter", name)
```